src/general/MainMenu.tscn

```diff
@@ -199,58 +200,41 @@ __meta__ = {
 
 [node name="LogoCenterContainer" type="CenterContainer" parent="MenuContainers"]
 anchor_left = 0.5
-anchor_top = 0.5
 anchor_right = 0.5
-anchor_bottom = 0.5
 margin_left = -640.0
-margin_top = -360.0
-margin_right = 640.0
-margin_bottom = -60.0
+margin_right = 639.0
+margin_bottom = 221.0
 __meta__ = {
 "_editor_description_": "Centering fo the Thrive logo"
 }
 
 [node name="Logo" type="TextureRect" parent="MenuContainers/LogoCenterContainer"]
 margin_left = 556.0
-margin_top = 66.0
+margin_top = 26.0
 margin_right = 723.0
-margin_bottom = 234.0
+margin_bottom = 194.0
 texture = ExtResource( 10 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Menus" type="Control" parent="MenuContainers"]
-anchor_top = 0.368
+[node name="Menus" type="CenterContainer" parent="MenuContainers"]
```

Comment:
This change is probably what's affecting the animation the most.

Reply:
I found it to make more sense to make it this way as I didn't understand how it was previously even working...

Also this should hopefully be the last button ever needed in the main menu, at least I hope, which is why I added it to the main menu. If we ever need more buttons, then a different design will be needed...

Reply:
We both agree on a big overhaul then.